lib/board.rb

```diff
@@ -0,0 +1,52 @@
+%w(board region square board_viewer).each { |file| require_relative file + '.rb' }
+class Board
+  attr_accessor :squares
+  attr_reader :regions, :size
+
+  # TODO: refactor
+  def initialize(size = 1)
+    @regions = []
+    @squares = []
+    @size = size
+  end
+
+  def populate
+    BoardMaker.new.populate_squares(self)
+    BoardMaker.new.populate_regions(self)
```

Comment:
Where is BoardMaker coming from? It's being required through Game, then called in Board, which couples it to both. Unless both classes need it, only require it in one